db/schema.sql

```diff
@@ -10029,13 +10077,6 @@ CREATE TABLE maker.osm_change (
 );
 
 
---
--- Name: TABLE osm_change; Type: COMMENT; Schema: maker; Owner: -
---
-
-COMMENT ON TABLE maker.osm_change IS 'Note event emitted when change is called on OSM/PIP contract(s)';
-
-
```

Comment:
Idk why this is showing up as part of the PR but seems like maybe a migration was updated without the schema getting a corresponding update as we removed comments?